RubeesEat/Model/DB/DbBillRepository.cs

```diff
@@ -110,7 +110,11 @@ public PaginatedView<BalanceChange> GetRecentBalanceChanges(Person currentUser,
                                                           SUM(el.Amount) AS TotalAmount, 
                                                           b.Date AS Date,
                                                           b.Description AS Description,
-                                                          b.BillID AS BillID
+                                                          b.BillID AS BillID,
```

Comment:
remove them while rebasing? 
Should we remove them, and then rebase on main (after the other branch is merged).

Reply:
Ok, i think i get it.
When rebasing, it will notfiy me that the same lines are changed in both files, and we should remove the changes from this branch